### PR TITLE
DOCS-3453: Convert FastCheckout SDKs to Readme

### PR DESCRIPTION
**Pull request by FrancescaNCook**
Do the changes meet MultiSafepay Docs' Definition of Done?
- Style guide, tone of voice, and naming conventions
- Tested
- Reviewed on mobile, tablet and desktop screens
- Notified stakeholders
- Pass GitHub Actions

### CODE REVIEW

File: content/developer/fastcheckout-android-sdk.md [hunk 1]
```diff
@@ -66,11 +129,12 @@ For how to set up each environment, see the example below.
    }
    ```
 
-5. Sync Gradle.
+4. Sync Gradle.
 
-You can now start using the Android SDK in your app.
+> ✅ Sucess
```
Comment (Sarah-MultiSafepay) [inline content/developer/fastcheckout-android-sdk.md:134]:
```suggestion
> ✅ Success
```

File: content/developer/fastcheckout-ios-sdk.md [hunk 2]
```diff
@@ -55,9 +118,10 @@ To learn more about creating, updating and retrieving orders, see the [API refer
 7. Under **Embedded binaries**,click **+**.
 8. Select **FastCheckoutKit.xcframework**.
 
-You can now start using the FastCheckoutKit SDK in your app.
+> ✅ Sucess
```
Comment (Sarah-MultiSafepay) [inline content/developer/fastcheckout-ios-sdk.md:121]:
```suggestion
> ✅ Success
```

File: content/developer/fastcheckout-ios-sdk.md [hunk 3]
```diff
@@ -81,9 +145,10 @@ manager.startCheckout(transactionId: "ID", host: self, onCompletion: { status, e
 })
 ```
 
-And that’s it! Now you can start a checkout and retrieve the <<glossary:transaction status>> once it completes.
+> ✅ Sucess
```
Comment (Sarah-MultiSafepay) [inline content/developer/fastcheckout-ios-sdk.md:148]:
```suggestion
> ✅ Success
```

**Review by Sarah-MultiSafepay — APPROVED** [review]
Just a couple of typos to fix.

File: content/developer/fastcheckout-android-sdk.md (hunk 1 above)
Comment (FrancescaNCook) [inline content/developer/fastcheckout-android-sdk.md:134]:
```suggestion
> ✅ Success
```

**Pull request merged**